plugin/storage/es/factory.go

```diff
@@ -272,6 +274,25 @@ func createSpanWriter(
 	return writer, nil
 }
 
+func (f *Factory) CreateSamplingStore(maxBuckets int) (samplingstore.Store, error) {
+	return createSamplingStore(f.getPrimaryClient, f.primaryConfig, f.logger)
+}
+
+func createSamplingStore(
+	clientFn func() es.Client,
+	cfg *config.Configuration,
+	logger *zap.Logger,
+) (samplingstore.Store, error) {
+	store := esSampling.NewSamplingStore(esSampling.SamplingStoreParams{
+		Client:          clientFn,
+		Logger:          logger,
+		IndexPrefix:     cfg.IndexPrefix,
+		IndexDateLayout: cfg.IndexDateLayoutDependencies,
```

Comment:
why "dependencies"?

Reply:
sorry, i should have checked it before pushing.